### ISSUE TITLE
Failed to get the hostname using the 'hostname' should not be a warning?

### ISSUE THREAD

**Issue by anuraaga** [line/armeria#1333]
I tried running an armeria server with the `gcr.io/distroless/java` docker image which does not include the `hostname` command. Is it too scary of an output?

```
WARN ] 2018-08-13 05:51:29.848 [main] com.linecorp.armeria.server.AbstractVirtualHostBuilder - Failed to get the hostname using the 'hostname' command; using InetAddress.getLocalHost() instead
java.io.IOException: Cannot run program "hostname": error=2, No such file or directory
	at java.lang.ProcessBuilder.start(ProcessBuilder.java:1048) ~[?:1.8.0_171]
	at java.lang.Runtime.exec(Runtime.java:620) ~[?:1.8.0_171]
	at java.lang.Runtime.exec(Runtime.java:450) ~[?:1.8.0_171]
	at java.lang.Runtime.exec(Runtime.java:347) ~[?:1.8.0_171]
	at com.linecorp.armeria.server.AbstractVirtualHostBuilder.<clinit>(AbstractVirtualHostBuilder.java:92) [armeria-0.65.1.jar:?]
	at com.linecorp.armeria.server.ServerBuilder.<init>(ServerBuilder.java:140) [armeria-0.65.1.jar:?]
	at org.curioswitch.common.server.framework.ServerModule.armeriaServer(ServerModule.java:334) [curio-server-framework-0.0.70.jar:?]
	at org.curioswitch.common.server.framework.ServerModule_ArmeriaServerFactory.proxyArmeriaServer(ServerModule_ArmeriaServerFactory.java:344) [curio-server-framework-0.0.70.jar:?]
	at org.curioswitch.common.server.framework.ServerModule_ArmeriaServerFactory.provideInstance(ServerModule_ArmeriaServerFactory.java:223) [curio-server-framework-0.0.70.jar:?]
	at org.curioswitch.common.server.framework.ServerModule_ArmeriaServerFactory.get(ServerModule_ArmeriaServerFactory.java:163) [curio-server-framework-0.0.70.jar:?]
	at org.curioswitch.common.server.framework.ServerModule_ArmeriaServerFactory.get(ServerModule_ArmeriaServerFactory.java:41) [curio-server-framework-0.0.70.jar:?]
	at dagger.internal.DoubleCheck.get(DoubleCheck.java:47) [dagger-2.16.jar:?]
	at com.stellarstation.api.fakeserver.DaggerFakeServerMain_FakeServerComponent.server(DaggerFakeServerMain_FakeServerComponent.java:322) [classes/:?]
	at com.stellarstation.api.fakeserver.FakeServerMain.main(FakeServerMain.java:63) [classes/:?]
Caused by: java.io.IOException: error=2, No such file or directory
	at java.lang.UNIXProcess.forkAndExec(Native Method) ~[?:1.8.0_171]
	at java.lang.UNIXProcess.<init>(UNIXProcess.java:247) ~[?:1.8.0_171]
	at java.lang.ProcessImpl.start(ProcessImpl.java:134) ~[?:1.8.0_171]
	at java.lang.ProcessBuilder.start(ProcessBuilder.java:1029) ~[?:1.8.0_171]
	... 13 more
```

**Comment by minwoox** [comment]
I think we probably need to lower the log level :)

**Comment by trustin** [comment]
Fixed via #1348
